packages/find-root/src/index.ts

```diff
@@ -20,34 +20,56 @@ import {
  * monorepo implementations first, with tools based on custom file schemas
  * checked last.
  */
-const defaultOrder: Tool[] = [
+export const DEFAULT_TOOLS = [
```

Comment:
I'd probably avoid exporting this for as long as it's not requested by the users. It's hard to predict how consumers use this and it could lock us from adding new items to this list.

Reply:
Do you have any particular code in mind that would utilize this?

Reply:
@Andarist By exporting the tool list I think it allows the caller to decide between two approaches:

```javascript
// explicitly limiting to my desired tools
findRootSync('.', [MyCustomTool, PnpmTool, YarnTool]);

// i just don't want to allow RootTool
findRootSync('.', DEFAULT_TOOLS.filter(tool => tool !== RootTool));
```

You can simulate the second option by just copying + pasting the list into your code and removing the ones you dislike; but, any future releases of tools won't automatically be used, you're on the hook to keep up-to-date.

Reply:
OK -- I'm willing to keep it internal for now, we can revisit if I end up with a definitive use case :)

Reply:
(I've made that change and also the other feedback.)